HW_2/task_2.py

```diff
@@ -0,0 +1,20 @@
+"""
+Напишите программу, которая принимает на вход натуральное число N и выдает список факториалов по основаниям от 1 до N
+
+Ввод: значение типа <int>
+Вывод: значение типа <list>
+
+Пример:
+4
+[1, 2, 6, 24]
+"""
+print('Введите число')
+n = int(input())
```

Comment:
Мы можем писать Сообщение для пользователя на приглашение к вводу внутри непосредственно функции `input`.
```suggestion
n = int(input('Введите число: '))
```